src/Parser.php

```diff
@@ -17,17 +17,18 @@ class Parser
     protected $parser;
 
     /**
-     * @param string $filePath Source file path
+     * @param string|mixed $input
      * @param callback|callable $itemCallback Callback
      * @param bool $assoc Parse as associative arrays
      *
      * @throws \Exception
      */
-    public function parse($filePath, $itemCallback, $assoc = true)
+    public function parse($input, $itemCallback, $assoc = true)
```

Comment:
This is breaking change. Let's introduce new method called `parseFromString`

Reply:
What does it break? It support both old and new inputs.